package.json

```diff
@@ -124,6 +124,7 @@
     "supertest": "^1.1.0",
     "swagger-tools": "^0.10.1",
     "throng": "^1.0.1",
+    "uglifyify": "^3.0.4",
```

Comment:
Awesome! I had been hung up on a version conflict with the "uglify" library so I thought we wouldn't be able to minify stuff until that got resolved.